runtime/trappist/src/lib.rs

```diff
@@ -616,18 +662,21 @@ construct_runtime!(
 
 		// Runtime features
 		Contracts: pallet_contracts = 40,
-		Council: pallet_collective::<Instance1> = 41,
 		Assets: pallet_assets = 42,
 		Identity: pallet_identity = 43,
 		Uniques: pallet_uniques = 44,
 		Scheduler: pallet_scheduler = 45,
 		Preimage: pallet_preimage = 46,
-		Democracy: pallet_democracy = 47,
 
 		// Handy utilities.
 		Utility: pallet_utility::{Pallet, Call, Event} = 50,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 51,
 
+		// Governance related
+		Council: pallet_collective::<Instance1> = 61,
```

Comment:
Ok for grouping Governance-related pallets together ... but why not start at 60 🥲️ (like in other groups) .. let's be consistent.